src/app/lib/config/services/portuguese.js

```diff
@@ -156,6 +157,7 @@ const service = {
       'BBC. A BBC não se responsabiliza pelo conteúdo de sites externos de internet',
   },
   fonts: [],
+  timezone: 'Europe/London',
```

Comment:
```suggestion
  timezone: 'America/Sao_Paulo',
```

Reply:
On the JIRA ticket WSFRONTPGS-168 I can see `Brasil` has a language of `pt`. I missed it because i thought there was a 1:1 mapping of service name to timezone. So my question would be is the `service` `Brasil` or `Portuguese`?